.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+Closes: <#issue>
+
+<!--- Describe your changes in detail -->
+
+## Checklist before requesting a review
+- [ ] [CHANGELOG.md](./CHANGELOG.md) updated
+- [ ] API changes are backwards-compatible
+- [ ] Workspace layout changes include a migration
+- [ ] Documentation updated
```

Comment:
Perhaps: `Documentation update PR: <link or N/A>` to force the person to link the docs changes

Reply:
Fixed via 5232927e7e596bafca6a35caae7754e7400490a2